exercises/practice/http-billion-dollar-question/modules/billionarehub_server/server.bal

```diff
@@ -0,0 +1,607 @@
+import ballerina/http;
+
+type Billionaire record {
+    string name;
+    float netWorth;
+};
+
+service / on new http:Listener(9090) {
+    resource function get getBillionaires(string country) returns Billionaire[]|error {
+        Billionaire[]? billionaires = topbillionaires[country];
+        if (billionaires is Billionaire[]) {
+            return billionaires;
+        }
+        return error(country + " is not found in the sample dataset.");
+    }
+}
+
+final map<Billionaire[]> topbillionaires = {
```

Comment:
Shall we use a csv to improve readability? Or is there a specific reason to use a map

Reply:
There is no specific reason, this file is not see by end users.
But anyway it is better to use csv approach.
Will update.

Reply:
Updated. Please check. dff21f3a570e21b4a892f362369d89c88b846425